pkg/controllers/awsloadbalancercontroller/deployment_test.go

```diff
@@ -562,10 +563,21 @@ func TestEnsureDeployment(t *testing.T) {
 				testContainer("controller", "test-image").withDefaultEnvs().withVolumeMounts(
 					corev1.VolumeMount{Name: "aws-credentials", MountPath: "/aws"},
 					corev1.VolumeMount{Name: "tls", MountPath: "/tls"},
+					corev1.VolumeMount{Name: "bound-sa-token", MountPath: "/var/run/secrets/openshift/serviceaccount", ReadOnly: true},
 				).build(),
 			).withControllerReference("cluster").withVolumes(
 				corev1.Volume{Name: "aws-credentials", VolumeSource: corev1.VolumeSource{Secret: &corev1.SecretVolumeSource{SecretName: "test-credentials"}}},
 				corev1.Volume{Name: "tls", VolumeSource: corev1.VolumeSource{Secret: &corev1.SecretVolumeSource{SecretName: "test-serving"}}},
+				corev1.Volume{Name: "bound-sa-token", VolumeSource: corev1.VolumeSource{Projected: &corev1.ProjectedVolumeSource{
+					DefaultMode: aws.Int32(420),
+					Sources: []corev1.VolumeProjection{{
+						ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+							Audience:          "openshift",
+							ExpirationSeconds: aws.Int64(3600),
```

Comment:
Change this please. Also do we have tests covering any change we make to the bound service account token?